ext/LandSimulationVisualizationExt.jl

```diff
@@ -275,6 +275,184 @@ function make_annual_timeseries(
     )
 end
 
+"""
+     make_diurnal_timeseries(
+        sim::ClimaLand.Simulations.LandSimulation;
+        savedir = ".",
+        short_names = nothing,
+	plot_name = "diurnal_timeseries.pdf",
+        comparison_data = nothing,
+        spinup_date = sim.start_date
```

Comment:
This is not currently stored in this location, but should be accessible from the simulation. Make that change in separate PR?

Reply:
oh, also note that since the fluxnet sims dont use the simulation object, they call the lower level make_*_timeseries plots for now